presto-main/src/test/java/io/prestosql/connector/MockConnectorFactory.java

```diff
@@ -194,6 +194,13 @@ private MockConnector(
             this.roleGrants = requireNonNull(roleGrants, "roleGrants is null");
         }
 
+        @Override
+        public boolean isSingleStatementWritesOnly()
```

Comment:
This should either be removed, or should return `true`, since the connector doesn't actually support writes. It is possible there are tests that rely on it returning `false`, but that's easy to verify.